src/data/themesData.jsx

```diff
@@ -0,0 +1,57 @@
+const themesData = [
+  {
+    themeImageURL: <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" fill="#7289da" className="w-20 h-20">
```

Comment:
Use separate `.svg` extension files for storing the image and use the relative URLs of the images instead. This is due to the fact that any changes to the actual code for the  image is unrelated to the themes data, so separating the inner details of image from actual matter should be done.

Reply:
Also after removal of the `<svg>` tag, the file extension should be changed back to `.js` as this does not contain any `jsx` syntax, rather simple data.